### ISSUE TITLE
Allow all countries

### ISSUE THREAD

**Issue by HenryAveMedi** [extcode/cart#414]
Is there an easy way to configure zones like
1 = DACH
2 = EU
3 = Rest of the world?

Or do i have to create them manually and put all of the hundreds of countries in the list?

And all the countries 1 by 1 in the allowedCountries { } ?

**Comment by rintisch** [comment]
@HenryAveMedi 
You can configure zones:
`plugin.tx_cart.shippings.zones`
`plugin.tx_cart.payments.zones`

Are you looking for something different?

**Comment by extcode** [comment]
https://docs.typo3.org/p/extcode/cart/main/en-us/Administrator/Configuration/PaymentMethods/Index.html#plugin-tx-cart-payments-zones

**Comment by rintisch** [comment]
I will close this ticket due to missing feedback. You're off course still welcome to respond!

**Comment by dahaupt** [comment]
@extcode @rintisch 
We have a similar use case for our project. All payment options should be allowed, independent of any country or zone. 

The select field in the cart simply shows all countries, we did not configure any countries using the TypoScript configuration. We created one zone for the payment methods. Unfortunately, a patch is necessary for this to work, as the configuration of `countries` is normally necessary within a zone.

Would be great to cover this simple case without specific country restrictions.
